### ISSUE TITLE
Cancelling the editor loses focus

### ISSUE THREAD

**Issue by dmaevsky** [fin-hypergrid/core#753]
Looks like after I press Escape while editing a cell, the keyboard navigation does not work any more until I click another cell with a mouse

**Comment by joneit** [comment]
Thanks for the heads up. I will look into this today.
